src/main/java/de/tum/in/www1/artemis/service/connectors/jenkins/dto/TestCaseDTO.java

```diff
@@ -33,4 +37,45 @@ public TestCaseDTO(String name, String classname, double time) {
     public boolean isSuccessful() {
         return CollectionUtils.isEmpty(errors) && CollectionUtils.isEmpty(failures);
     }
+
+    @Override
+    public String getName() {
+        return name;
+    }
+
+    @Override
+    public List<String> getMessage() {
+        return extractMessage().map(Collections::singletonList).orElse(Collections.emptyList());
+    }
+
+    private Optional<String> extractMessage() {
```

Comment:
Copy the comment from the original method. Otherwise, the name does not tell you, what the method really do :)